var/spack/repos/builtin/packages/h5bench/package.py

```diff
@@ -38,6 +41,7 @@ class H5bench(CMakePackage):
 
     depends_on("cmake@3.10:", type="build")
     depends_on("mpi")
+    depends_on("slurm")
```

Comment:
This is only needed for testing so at this point you can limit this to a `run` dependency.

```suggestion
    depends_on("slurm", type="run")
```